actix-session/Cargo.toml

```
@@ -48,16 +49,21 @@ futures-core = { version = "0.3.7", default-features = false, optional = true }
 # redis-rs-session
 redis = { version = "0.21", default-features = false, features = ["aio", "tokio-comp", "connection-manager"], optional = true }
 
+# sqlite-session
+rusqlite = { version = "0.27", features = ["bundled", "chrono"], optional = true }
+r2d2 = { version = "0.8", optional = true }
+r2d2_sqlite = { version = "0.20", optional = true }
+
 [dev-dependencies]
-actix-session = { path = ".", features = ["cookie-session", "redis-actor-session", "redis-rs-session"] }
+actix-session = { path = ".", features = ["cookie-session", "sqlite-session"] }
```

```suggestion
actix-session = { path = ".", features = ["cookie-session", "redis-actor-session", "sqlite-session"] }
```
We should keep enabling all features for the test suite, that's where your error is coming from.